samples/cpp/speech_sample/utils.hpp

```diff
@@ -535,3 +535,37 @@ std::map<std::string, std::string> parse_input_layouts(const std::string& layout
         throw std::logic_error("Can't parse input parameter string: " + layout_string);
     return return_value;
 }
+
+/**
+ * @brief Parse parameters for inputs/outputs like as "<input_name1>=<file1.ark/.npz>,<input_name2>=<file2.ark/.npz>" or
+ * "<file.ark/.npz>" in case of one input
+ * @param file_paths_string input path
+ * @return pair of filename and vector of inputNameBlobs
+ */
+std::pair<std::string, std::vector<std::string>> parse_parameters(const std::string file_paths_string) {
+    auto search_string = file_paths_string;
+    char comma_delim = ',';
+    char equal_delim = '=';
+    std::string filename = "";
+    std::vector<std::string> inputNameBlobs;
```

Comment:
Let's align code style for new code at least. And it seems that function is common for inputs and outputs.
```suggestion
    std::vector<std::string> tensor_names;
```